### PR TITLE
Add brief details about and link to, windbg's .kdfiles command (#1493)

### PR DESCRIPTION
**Pull request by dv-msft**
## Description

Add some verbiage for WinDbg's ```.kdfiles``` command.

## Testing

Doc update only - No testing required.

## Documentation

```GettingStarted.md``` was updated.

Resolves: #1493

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -358,3 +358,10 @@ and checked the Development checkbox, installation was completed for you.
 Otherwise, after installing the nuget package, as a one-time operation, you will
 currently need to run the `export_program_info.exe` tool to complete the install. This
 tool can be found in your project's `packages\eBPF-for-Windows\build\native\bin` directory.
+
+If you are working on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This
```
Comment (dthaler) [inline docs/GettingStarted.md:362]: Does this also apply when using the VS [Debugger Immediate Window](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/entering-debugger-commands-in-visual-studio)?  If not, then update the first part of the sentence, as in:

```suggestion
If you are using WinDbg to work on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This
```

If kdfiles does in fact work in the VS debugger immediate window, then update latter half of the sentence to not be limited to WinDbg.

Comment (dv-msft) [reply docs/GettingStarted.md]: Driver debugging using VS requires the installation of the WDK on the test device [(for VS intergration).](https://learn.microsoft.com/en-us/windows-hardware/drivers/gettingstarted/provision-a-target-computer-wdk-8-1).  Unfortunately, 'Note' @ the [WDK install page](https://learn.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk) mentions that driver debugging under VS is currently non-functional and instead recommends using WinDbg instead.

I've updated this verbiage accordingly.

**Review by Alan-Jowett — DISMISSED** [review]

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -358,3 +358,10 @@ and checked the Development checkbox, installation was completed for you.
 Otherwise, after installing the nuget package, as a one-time operation, you will
 currently need to run the `export_program_info.exe` tool to complete the install. This
 tool can be found in your project's `packages\eBPF-for-Windows\build\native\bin` directory.
+
+If you are using WinDbg to work on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This
+command allows the replacement of a driver binary on the target machine with another binary from the machine WinDbg is running
+on (typically the development machine) at driver load time.
+
+This eliminates the need for repeated manual copy of the modified driver binary and saves considerable time during the
+development cycle.  More details are [here](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/-kdfiles--set-driver-replacement-map-)
```
Comment (dthaler) [inline docs/GettingStarted.md:367]:
```suggestion
development cycle.  See the [Windows Hardware Developer documentation](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/-kdfiles--set-driver-replacement-map-) for more details.
```

Explanation: never use "here" as link text, as it violates [W3C guidelines](https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text) which are basically for accessibility (screen readers, etc.).

Comment (dv-msft) [reply docs/GettingStarted.md]: Huh. Had never thought of that.  Thanks for pointing this out! Fixed.

**Review by dthaler — APPROVED** [review]

**Review by saxena-anurag — APPROVED** [review]

**Pull request merged**